SPECS/systemd/systemd.spec

```diff
@@ -18,6 +18,8 @@ Patch1:         add-fsync-sysusers-passwd.patch
 # Patch2 can be removed once we update systemd to a version containing the following commit:
 # https://github.com/systemd/systemd/commit/d5cb053cd93d516f516e0b748271b55f9dfb3a29
 Patch2:         gpt-auto-devno-not-determined.patch
+# Patch3 can be removed once we update to major version 251 or higher:
+Patch3:         CVE-2022-3821.patch
```

Comment:
To keep the specs as close to each other as possible, could you also patch `systemd-bootstrap`?

Reply:
systemd-bootstrap.spec has been updated to have the same patch added